### PR TITLE
LTD-215: Reporting - Add query for standard licences with goods and ratings

### PR DESCRIPTION
**Pull request by r4vi**
To act as a template for building reports that aggregate this raw data like
summary of SIELs

### CODE REVIEW

File: api/reports/queries/standard.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+GOODS_AND_RATINGS = """
+WITH goods as (
+    select ll.reference_code         "licence_ref"
+         , sa.baseapplication_ptr_id "appliaction_id"
```
Comment (kerin) [inline api/reports/queries/standard.py:4]: typo?

Comment (saruniitr) [inline api/reports/queries/standard.py:4]: typo in `application_id`

**Review by kerin — COMMENTED** [review]
Is it worth thinking about adding tests at this stage, or is this proof of concept -ish?

File: api/reports/queries/standard.py (hunk 1 above)
Comment (r4vi) [reply api/reports/queries/standard.py]: thanks - good spot

**Review by kerin — APPROVED** [review]

**Pull request merged**